src/vm/compilestubs.cpp

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
(I believe similar approach can be used to make the ifdefs around EEJitManager cleaner - ifdef out the whole class.)